app/lib/promHistograms.js

```diff
@@ -2,6 +2,7 @@ const promClient = require('./promBundle').promClient;
 const buckets = require('./constants').promHistogramBuckets;
 
 module.exports = {
-  esGetPharmacy: new promClient.Histogram({ name: 'es_get_pharmacy', help: 'Duration histogram of Elasticsearch request to get Pharmacies', buckets }),
-  filterServices: new promClient.Histogram({ name: 'filter_services', help: 'Duration histogram of filtering the returned pharmacies based on their openness', buckets }),
+  esGetNearbyPharmacy: new promClient.Histogram({ name: 'es_get_nearby_pharmacy', help: 'Duration histogram of Elasticsearch request to get nearby Pharmacies', buckets }),
+  esGetOpenPharmacy: new promClient.Histogram({ name: 'es_get_open_pharmacy', help: 'Duration histogram of Elasticsearch request to get open Pharmacies', buckets }),
+  esGetTotalPharmacy: new promClient.Histogram({ name: 'es_get_total_pharmacy', help: 'Duration histogram of sum of both Elasticsearch requests to get Pharmacies', buckets }),
```

Comment:
In the future, if the (current) two requests are combined the help message isn't going to make too much sense. Suggest the name is changed to `es_get_pharmacy` (also, should this be pluralised? and if so the others should also). Suggest the help is changed to `Duration histogram of Elasticsearch request to get Pharmacies`.